lisa/sut_orchestrator/azure/features.py

```diff
@@ -1512,8 +1510,11 @@ def create_setting(
             "standardMSv2Family",
         ]:
             # https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#how-can-i-find-vm-sizes-that-support-trusted-launch # noqa: E501
-            tvm_disable_value = raw_capabilities.get("TrustedLaunchDisabled", "False")
-            if gen_value and ("V2" in str(gen_value)) and (not eval(tvm_disable_value)):
+            if (
+                gen_value
+                and ("V2" in str(gen_value))
+                and raw_capabilities.get("TrustedLaunchDisabled", "False") == "True"
```

Comment:
is it `raw_capabilities.get("TrustedLaunchDisabled", "False") == "False"`

Reply:
Yes, good catch!